### ISSUE TITLE
eRL_demo_PPOinSingleFile.py has wrong sign for the entropy term in the loss function

### ISSUE THREAD

**Issue by Daffan** [AI4Finance-Foundation/ElegantRL#74]
In this implementation, ElegantRL/elegantrl_helloworld/eRL_demo_PPOinSingleFile.py, the sign for the entropy term in the loss function is wrong.

**Comment by Yonv1943** [comment]
The original code in `eRL_demo_PPOinSingleFile.py` is correct.

See [line43 the negative sign in `logprob`](https://github.com/AI4Finance-Foundation/ElegantRL/blob/177c6e12dfa29d9d5a2d78f3eeebddf5c91404d4/elegantrl_helloworld/eRL_demo_PPOinSingleFile.py#L43)
```
class ActorPPO(nn.Module):
    ...
    def get_logprob_entropy(self, state, action):
        ...
        logprob = -(self.a_logstd + self.sqrt_2pi_log + delta).sum(1)  # new_logprob
        #         ↑ the negative sign
        ...
```


See [line 188 the entropy term `obj_entropy`](https://github.com/AI4Finance-Foundation/ElegantRL/blob/177c6e12dfa29d9d5a2d78f3eeebddf5c91404d4/elegantrl_helloworld/eRL_demo_PPOinSingleFile.py#L188) 

```
class AgentPPO:
    ...
    def update_net(self, buffer, batch_size, repeat_times, soft_update_tau):
        ...
        for _ in range(int(buf_len / batch_size * repeat_times)):
            ...
            new_logprob, obj_entropy = self.act.get_logprob_entropy(state, action)  # it is obj_actor
            ...
            obj_actor = obj_surrogate + obj_entropy * self.lambda_entropy
            self.optim_update(self.act_optim, obj_actor)
            # minimize objective

    def optim_update(optimizer, objective):
        optimizer.zero_grad()
        objective.backward()
        optimizer.step()
```

**Comment by XiaoYangLiu-FinRL** [comment]
We discussed this, the sign is correct;  double negative leads to a positive sign.
